test/integration/controller/jobs/mxjob/mxjob_controller_test.go

```diff
@@ -374,27 +281,31 @@ var _ = ginkgo.Describe("Job controller interacting with scheduler", ginkgo.Orde
 	})
 
 	ginkgo.It("Should schedule jobs as they fit in their ClusterQueue", func() {
-		ginkgo.By("creating localQueue")
```

Comment:
This comment would be worth it in all kfjobs.

Reply:
added https://github.com/kubernetes-sigs/kueue/pull/1310/commits/4c54a9f1c43c133e8ee433a7364f8aa7ba51415e